### PR TITLE
(new): introducing custom attributes (fixes #287)

### LINKED ISSUE #287: (Re-)Implement support for custom attributes?
With #247, fragment attributes have been trimmed down to a selection of default properties and thus support for custom attributes has been removed. We’re using v3.7.1 in our project (the last version supporting the old attributes handling) as we liked to implement custom properties and take those into consideration within a custom `requestFragment` handler.

Is there any plan to (re-)implement support for custom attributes? For example by specifying the allowed custom properties within the setup options.

### PR DESCRIPTION
**Pull request by morriq**
#287

### CODE REVIEW

File: lib/fragment.js [hunk 1]
```diff
@@ -62,6 +64,10 @@ const getAttributes = (tag, context) => {
         fallbackUrl: attributes['fallback-src'],
         timeout: parseInt(timeout || 3000, 10)
     };
+
+    delete finalAttributes['fallback-src'];
```
Comment (vigneshshanmugam) [inline lib/fragment.js]: this would trigger deopt, why do you need to delete this property?

Comment (morriq) [reply lib/fragment.js]: good point. fixed in https://github.com/zalando/tailor/pull/323/commits/920f6e2a51678c3e73a929fc60e3fc5f1af15e6f .

**Comment by vigneshshanmugam** [comment]
👍

**Comment by addityasingh** [comment]
👍

**Comment by dominiklaube** [comment]
@boopathi and @ruiaraujo it would be great, if you please could review this pull request :)

**Comment by pmoisel** [comment]
Is there a chance for getting this PR merged?

**Comment by pmoisel** [comment]
Can we expect this to be merged? We would like to update. Since we use custom attributes we can not

**Comment by pmoisel** [comment]
Hi @vigneshshanmugam 
Thanks for merging this. Is there a chance to get a new release from master? :)

**Pull request merged**